src/app/(admin)/admin/_components/ServiceProvider/ContactsForm.js

```diff
@@ -0,0 +1,30 @@
+import { TextInput } from 'react-admin';
+import { FormFieldWrapper } from '../FormFieldWrapper';
+
+const fieldGroupClass = 'flex w-full flex-col md:flex-row md:gap-6';
+
+export function ContactsForm() {
+  return (
+    <FormFieldWrapper title={'Контакти'} className="mt-3 w-full">
```

Comment:
It is worth using one approach to writing string props, so consider using rules:
[eslint-plugin-react](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-curly-brace-presence.md)
[jsx-quotes](https://eslint.org/docs/latest/rules/jsx-quotes)

Reply:
Thanks for comment! Defined `'react/jsx-curly-brace-presence': [2, { props: 'never', children: 'never' }]` to highlight unnecessary braces.